python/python/lance/dataset.py

```diff
@@ -1321,6 +1332,17 @@ def create_index(
                 num_sub_vectors=16
             )
 
+        .. code-block:: python
+
+            import lance
+
+            dataset = lance.dataset("/tmp/sift.lance")
+            dataset.create_index(
+                "vector",
+                "IVF_HNSW_SQ",
```

Comment:
Do we have an explanation about what SQ is?

Reply:
just added